src/components/Badge.tsx

```diff
@@ -33,40 +31,13 @@ interface BadgeProps {
 let Badge = ({ content, className, amount, prefix }: BadgeProps) => (
   <div className={className}>
     {prefix && <span>{prefix}</span>}
-    {amount && content ? <Amount value={BigInt(content)} /> : content}
+    {amount && content ? <Amount value={BigInt(content)} fadeDecimals /> : content}
   </div>
 )
 
-const getBadgeColor = (badgeType: BadgeType, theme: DefaultTheme) => {
-  let backgroundColor
-  let color
-
-  switch (badgeType) {
-    case 'plus':
-      backgroundColor = 'rgba(48, 167, 84, 0.12)'
-      color = '#37c461'
-      break
-    case 'minus':
-      backgroundColor = 'rgba(243, 113, 93, 0.1)'
-      color = 'rgba(243, 113, 93, 1)'
-      break
-    case 'neutral':
-      backgroundColor = theme.name === 'dark' ? 'rgba(58, 58, 58, 0.28)' : 'rgba(90, 90, 90, 0.6)'
-      color = theme.name === 'dark' ? 'rgba(255, 255, 255, 0.8)' : 'rgba(255, 255, 255, 1)'
-      break
-    case 'neutralHighlight':
-      backgroundColor = 'rgba(101, 16, 247, 1)'
-      color = 'rgba(255, 255, 255, 1)'
-  }
-
-  return { backgroundColor, color }
-}
-
 Badge = styled(Badge)`
-  background-color: ${({ type, theme }) => getBadgeColor(type, theme).backgroundColor};
-  color: ${({ type, theme }) => getBadgeColor(type, theme).color};
+  color: ${({ type, theme }) => theme.badge.font[type]};
```

Comment:
A badge without a background color isn't really a badge anymore I would say, but just a stylised string.
Your `DarkLabel` (used for from/to labels) looks more like a badge.

That's a purely style related discussion we should have. In the explorer, I'm using badges for amounts in tables + many other things. In the wallet, I tried another approach to make the UI feel less cluttered (we will have more content to display than in the explorer). Which approach do you prefer?

IMO I tend to think that badges should be used carefully for things we wanna highlight or label, be it for address labels, important status, total amounts etc.. Badges can be with or without stroke, depending on the visual importance we wanna give.

We would keep repeating amounts (in table) or other inline data as simple strings (and make the change in the explorer as well).

As a side note, I'd like to reduce a bit the use of colors accros my mockups (for instance, not use green and red for every amount diff in a table). I'll experiment around that.

**TLDR: Let's keep the badge component, but let's not use it for TX "direction" label or table amounts. :) And I need to keep iterating.**

Reply:
I'm making changes to the mockups right now, so don't spend too much time on this. This was mostly to keep you up to date with my thoughts 🧠

Reply:
Thanks for sharing your thoughts. I agree with all of them.